plugins/modules/storage/zfs/zfs_delegate_admin.py

```diff
@@ -52,7 +52,6 @@
     description:
       - The list of permission(s) to delegate (required if C(state) is C(present)).
```

Comment:
How about adding the current list of properties as examples here:
```suggestion
      - The list of permission(s) to delegate (required if C(state) is C(present)).
      - Permissions are for example C(allow), C(clone), C(create), C(destroy), C(diff), C(hold), C(mount), C(promote), C(readonly), C(receive), C(release), C(rename), C(rollback), C(send), C(share), C(snapshot), C(unallow).
```

Reply:
I feel like it's not a great idea to list these, but I've added a link to OpenZFS documentation here for convenience. There are some actual examples in the examples section which show some possible permission values already.

Reply:
That's also fine for me.